.prettierrc

```
@@ -4,5 +4,5 @@
   "tabWidth": 2,
   "semi": true,
   "singleQuote": true,
-  "endOfLine": "lf"
+  "endOfLine": "crlf"
```

can you remake this line like this before committing?
```suggestion
  "endOfLine": "lf"
```